src/core/client/stream/tabs/Comments/Comment/CommentContainer.tsx

```diff
@@ -622,8 +658,30 @@ export const CommentContainer: FunctionComponent<Props> = ({
                             </Localized>
                           </Flex>
                         </Button>
+                        <Button
+                          color="stream"
+                          variant="text"
+                          onClick={() => openEditDialog('DELETED')}
+                          className={cn(
+                            CLASSES.comment.topBar.editButton,
+                            styles.editButton
+                          )}
+                          data-testid="comment-delete-button"
+                        >
+                          <Flex alignItems="center" justifyContent="center">
+                            <SvgIcon
+                              Icon={DeleteIcon}
+                              className={styles.editIcon}
+                            />
+                            <Localized id="comments-commentContainer-deleteButton">
+                              Delete
+                            </Localized>
+                          </Flex>
+                        </Button>
+                        
+</>
                       )}
-                      {showAvatar && comment.author?.avatar && (
```

Comment:
Please don't disable feature flag options in our code-base. They were put there intentionally. If you want to show avatars on your Coral instance, please enable the feature flag via the API.